Broken PR. Will be deleted (#9924)
ydb-platform/ydb: ydb/docs/en/core/dev/troubleshooting/troubleshooting-performance-issues.md

# Troubleshooting performance issues

## Overview

> Addressing database performance issues often requires a holistic approach, including optimizing queries, configuring hardware resources correctly, and ensuring that both the database and application are well-designed. Regular monitoring and maintenance are key to identifying and resolving these issues proactively.

Reviewer: ```suggestion
Addressing database performance issues often requires a holistic approach, which includes optimizing queries, properly configuring hardware resources, and ensuring that both the database and the application are well-designed. Regular monitoring and maintenance are essential for proactively identifying and resolving these issues.
```

## Tools to troubleshoot performance issues

- {{ ydb-short-name }} metrics

- {{ ydb-short-name }} logs

> - {{ ydb-short-name }} CLI

Reviewer: Add links
